Test: UninstallFromPMCForPC_SucceedAsync is flaky

The test `NuGet.Tests.Apex.AuthorSignedPackageTestCase.UninstallFromPMCForPC_SucceedAsync(projectTemplate: ClassLibrary)` has timed out on at least two builds: [1](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3669401&view=logs&j=54d34e6c-10d1-5f6f-0d73-85c1fa19385b&t=2bc2bfed-4a11-5a5c-814b-223591eb3d53), [2](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3672284&view=logs&j=54d34e6c-10d1-5f6f-0d73-85c1fa19385b&t=2bc2bfed-4a11-5a5c-814b-223591eb3d53)

Since it times out, the test results xml doesn't get generated, which means Azure DevOps test result analysis can't help us determine how often it's happening (the Apex tests job has timed out for multiple reasons in the past)

```
     NuGet.Tests.Apex.AuthorSignedPackageTestCase.UninstallFromPMCForPC_SucceedAsync(projectTemplate: ClassLibrary) [STARTING]
     NuGet.Tests.Apex: [Long Running Test] 'NuGet.Tests.Apex.AuthorSignedPackageTestCase.UninstallFromPMCForPC_SucceedAsync', Elapsed: 00:05:04
     NuGet.Tests.Apex: [Long Running Test] 'NuGet.Tests.Apex.AuthorSignedPackageTestCase.UninstallFromPMCForPC_SucceedAsync', Elapsed: 00:10:04
     NuGet.Tests.Apex: [Long Running Test] 'NuGet.Tests.Apex.AuthorSignedPackageTestCase.UninstallFromPMCForPC_SucceedAsync', Elapsed: 00:15:05
     NuGet.Tests.Apex: [Long Running Test] 'NuGet.Tests.Apex.AuthorSignedPackageTestCase.UninstallFromPMCForPC_SucceedAsync', Elapsed: 00:20:05
     NuGet.Tests.Apex: [Long Running Test] 'NuGet.Tests.Apex.AuthorSignedPackageTestCase.UninstallFromPMCForPC_SucceedAsync', Elapsed: 00:25:05
     NuGet.Tests.Apex: [Long Running Test] 'NuGet.Tests.Apex.AuthorSignedPackageTestCase.UninstallFromPMCForPC_SucceedAsync', Elapsed: 00:30:05
     NuGet.Tests.Apex: [Long Running Test] 'NuGet.Tests.Apex.AuthorSignedPackageTestCase.UninstallFromPMCForPC_SucceedAsync', Elapsed: 00:35:05
Attempting to cancel the build...
##[error]The task has timed out.
```

--- a/test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/IVsServicesTestCase.cs
+++ b/test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/IVsServicesTestCase.cs
@@ -45,7 +45,7 @@ namespace NuGet.Tests.Apex
             CommonUtility.AssertPackageInPackagesConfig(VisualStudio, projExt, "newtonsoft.json", XunitLogger);
         }
 
-        [StaFact(Skip = "https://github.com/NuGet/Home/issues/11308")]
+        [StaFact]
         public void SimpleUninstallFromIVsInstaller()
         {
             // Arrange
@@ -82,7 +82,7 @@ namespace NuGet.Tests.Apex
             userPackagesFolder.Should().NotBe(_pathContext.UserPackagesFolder);
         }
 
-        [StaFact(Skip = "https://github.com/NuGet/Home/issues/11308")]
+        [StaFact]
         public async Task SimpleInstallFromIVsInstaller_PackageSourceMapping_WithSingleFeed()
         {
             // Arrange
@@ -133,7 +133,7 @@ namespace NuGet.Tests.Apex
             File.Exists(uniqueContentFile).Should().BeTrue($"'{uniqueContentFile}' should exist");
         }
 
-        [StaFact(Skip = "https://github.com/NuGet/Home/issues/11308")]
+        [StaFact]
         public async Task SimpleInstallFromIVsInstaller_PackageSourceMapping_WithMultipleFeedsWithIdenticalPackages_InstallsCorrectPackage()
         {
             // Arrange

--- a/test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NetCoreProjectTestCase.cs
+++ b/test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NetCoreProjectTestCase.cs
@@ -121,7 +121,7 @@ namespace NuGet.Tests.Apex
             }
         }
 
-        [NuGetWpfTheory]
+        [NuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/12898")]
         [MemberData(nameof(GetNetCoreTemplates))]
         public async Task WithSourceMappingEnabled_InstallAndUpdatePackageFromPMUIFromExpectedSource_Succeeds(ProjectTemplate projectTemplate)
         {

--- a/test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NetCoreProjectTestCase.cs
+++ b/test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NetCoreProjectTestCase.cs
@@ -17,7 +17,7 @@ namespace NuGet.Tests.Apex
         }
 
         // basic create for .net core template
-        [NuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/11308")]
+        [NuGetWpfTheory]
         [MemberData(nameof(GetNetCoreTemplates))]
         public void CreateNetCoreProject_RestoresNewProject(ProjectTemplate projectTemplate)
         {
@@ -31,7 +31,7 @@ namespace NuGet.Tests.Apex
         }
 
         // basic create for .net core template
-        [NuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/9410")]
+        [NuGetWpfTheory]
         [MemberData(nameof(GetNetCoreTemplates))]
         public void CreateNetCoreProject_AddProjectReference(ProjectTemplate projectTemplate)
         {
@@ -121,7 +121,7 @@ namespace NuGet.Tests.Apex
             }
         }
 
-        [NuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/11308")]
+        [NuGetWpfTheory]
         [MemberData(nameof(GetNetCoreTemplates))]
         public async Task WithSourceMappingEnabled_InstallAndUpdatePackageFromPMUIFromExpectedSource_Succeeds(ProjectTemplate projectTemplate)
         {

--- a/test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetConsoleTestCase.cs
+++ b/test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetConsoleTestCase.cs
@@ -21,7 +21,7 @@ namespace NuGet.Tests.Apex
         {
         }
 
-        [NuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/11308")]
+        [NuGetWpfTheory]
         [MemberData(nameof(GetPackageReferenceTemplates))]
         public async Task InstallPackageFromPMCWithNoAutoRestoreVerifyAssetsFileAsync(ProjectTemplate projectTemplate)
         {
@@ -199,7 +199,7 @@ namespace NuGet.Tests.Apex
             }
         }
 
-        [NuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/8469")]
+        [NuGetWpfTheory]
         [MemberData(nameof(GetNetCoreTemplates))]
         public async Task NetCoreTransitivePackageReferenceLimitAsync(ProjectTemplate projectTemplate)
         {
@@ -242,7 +242,7 @@ namespace NuGet.Tests.Apex
             }
         }
 
-        [NuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/8386")]
+        [NuGetWpfTheory]
         [InlineData(ProjectTemplate.ClassLibrary, false)]
         [InlineData(ProjectTemplate.NetCoreConsoleApp, true)]
         [InlineData(ProjectTemplate.NetStandardClassLib, true)]
@@ -293,7 +293,7 @@ namespace NuGet.Tests.Apex
             }
         }
 
-        [NuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/11308")]
+        [NuGetWpfTheory]
         [MemberData(nameof(GetPackagesConfigTemplates))]
         public async Task InstallPackageForPC_PackageSourceMapping_WithSingleFeed(ProjectTemplate projectTemplate)
         {
@@ -338,7 +338,7 @@ namespace NuGet.Tests.Apex
             }
         }
 
-        [NuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/11308")]
+        [NuGetWpfTheory]
         [MemberData(nameof(GetPackagesConfigTemplates))]
         public async Task UpdatePackageForPC_PackageSourceMapping_WithSingleFeed(ProjectTemplate projectTemplate)
         {
@@ -386,7 +386,7 @@ namespace NuGet.Tests.Apex
             }
         }
 
-        [NuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/11308")]
+        [NuGetWpfTheory]
         [MemberData(nameof(GetPackagesConfigTemplates))]
         public async Task InstallPackageForPC_PackageSourceMapping_WithMultipleFeedsWithIdenticalPackages_InstallsCorrectPackage(ProjectTemplate projectTemplate)
         {
@@ -449,7 +449,7 @@ namespace NuGet.Tests.Apex
             }
         }
 
-        [NuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/11308")]
+        [NuGetWpfTheory]
         [MemberData(nameof(GetPackagesConfigTemplates))]
         public async Task UpdatePackageForPC_PackageSourceMapping_WithMultipleFeedsWithIdenticalPackages_UpdatesCorrectPackage(ProjectTemplate projectTemplate)
         {
@@ -513,7 +513,7 @@ namespace NuGet.Tests.Apex
             }
         }
 
-        [NuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/8386")]
+        [NuGetWpfTheory]
         [InlineData(ProjectTemplate.ClassLibrary, false)]
         [InlineData(ProjectTemplate.NetStandardClassLib, true)]
         public async Task UpdateAllReinstall_WithPackageReferenceProject_WarnsAsync(ProjectTemplate projectTemplate, bool warns)
@@ -610,7 +610,7 @@ namespace NuGet.Tests.Apex
             Assert.True(VisualStudio.HasNoErrorsInOutputWindows());
         }
 
-        [NuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/11308")]
+        [NuGetWpfTheory]
         [MemberData(nameof(GetPackageReferenceTemplates))]
         public async Task UpdatePackageForPR_PackageNamespace_WithMultipleFeedsWithIdenticalPackages_InstallsCorrectPackage(ProjectTemplate projectTemplate)
         {

--- a/test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetConsoleTestCase.cs
+++ b/test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetConsoleTestCase.cs
@@ -513,7 +513,7 @@ namespace NuGet.Tests.Apex
             }
         }
 
-        [NuGetWpfTheory]
+        [NuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/12899")]
         [InlineData(ProjectTemplate.ClassLibrary, false)]
         [InlineData(ProjectTemplate.NetStandardClassLib, true)]
         public async Task UpdateAllReinstall_WithPackageReferenceProject_WarnsAsync(ProjectTemplate projectTemplate, bool warns)

--- a/test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetPackageSigningTests/AuthorSignedPackageTestCase.cs
+++ b/test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetPackageSigningTests/AuthorSignedPackageTestCase.cs
@@ -45,7 +45,7 @@ namespace NuGet.Tests.Apex
             }
         }
 
-        [CIOnlyNuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/9476")]
+        [CIOnlyNuGetWpfTheory]
         [MemberData(nameof(GetPackagesConfigTemplates))]
         public async Task UninstallFromPMCForPC_SucceedAsync(ProjectTemplate projectTemplate)
         {
@@ -67,7 +67,7 @@ namespace NuGet.Tests.Apex
             }
         }
 
-        [CIOnlyNuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/11308")]
+        [CIOnlyNuGetWpfTheory]
         [MemberData(nameof(GetPackagesConfigTemplates))]
         public async Task UpdateUnsignedToSignedVersionFromPMCForPC_SucceedAsync(ProjectTemplate projectTemplate)
         {
@@ -91,7 +91,7 @@ namespace NuGet.Tests.Apex
             }
         }
 
-        [CIOnlyNuGetWpfTheory(Skip = "https://github.com/NuGet/Home/issues/11308")]
+        [CIOnlyNuGetWpfTheory]
         [MemberData(nameof(GetPackagesConfigTemplates))]
         public async Task DowngradeSignedToUnsignedVersionFromPMCForPC_SucceedAsync(ProjectTemplate projectTemplate)
         {

--- a/test/NuGet.Tests.Apex/NuGet.Tests.Apex/xunit.runner.json
+++ b/test/NuGet.Tests.Apex/NuGet.Tests.Apex/xunit.runner.json
@@ -2,5 +2,6 @@
   "methodDisplay": "classAndMethod",
   "longRunningTestSeconds": 300,
   "diagnosticMessages": true,
-  "preEnumerateTheories": false
+  "preEnumerateTheories": false,
+  "shadowCopy": false
 }

